griptape/engines/image_generation/image_generation_engine.py

```diff
@@ -0,0 +1,31 @@
+from typing import Optional
+
+from attr import field, define
+from griptape.drivers import BaseImageGenerationDriver
+from griptape.rules import Rule, Ruleset
+
+
+@define
+class ImageGenerationEngine:
+    image_generation_driver: BaseImageGenerationDriver = field(kw_only=True)
+
+    def generate_image(
+        self,
+        prompts: list[str],
+        negative_prompts: Optional[list[str]] = None,
+        rulesets: Optional[list[Ruleset]] = None,
+        negative_rulesets: Optional[list[Ruleset]] = None,
+        **kwargs
```

Comment:
Do we need `kwargs`? As far as I can tell it's not being used anywhere.

Reply:
That was needed when additional parameters came through the method call and not the driver setup. Removed.